### PR TITLE
adds new config items for mattermost 4.3

### PR DESCRIPTION
**Pull request by tejaycar**

### CODE REVIEW

File: packages/load_test/packaging [hunk 1]
```diff
@@ -6,4 +6,4 @@ set -u # report the usage of uninitialized variables
 # $BOSH_INSTALL_TARGET - where you copy/install files to be included in package
 
 tar xzf mattermost/load-test-*.tar.gz
-cp -a load-test*/bin $BOSH_INSTALL_TARGET
+cp -a mattermost-load-test*/bin $BOSH_INSTALL_TARGET
```
Comment (drnic) [inline packages/load_test/packaging]: Strange. This suggests we're not actually running/compiling the `load_test` package in CI. I'll investigate this.

**Comment by drnic** [comment]
@tejaycar looks good!

**Comment by tejaycar** [comment]
@drnic  Hmm... merging this guy in doesn't appear to have created a new bosh release either.  I assume that should trigger automatically?  Or am I jumping the gun and it's just in flight still?

**Comment by drnic** [comment]
The “shipit” job in CI is still manual. I’ll press it soon (after breakfast).

I do want to find a way to share access to the community CI pipelines so you could run “shipit”. It’s on my todo list.

________________________________

**Comment by drnic** [comment]
@tejaycar release cut!

**Pull request merged**